geoalchemy2/types.py

```diff
@@ -129,7 +132,7 @@ class _GISType(UserDefinedType):
         geometry/geography columns. """
 
     def __init__(self, geometry_type='GEOMETRY', srid=-1, dimension=2,
-                 spatial_index=True, management=False, use_typmod=None,
+                 spatial_index=True, is_N_D_index=False, management=False, use_typmod=None,
```

Comment:
What about `use_N_D_index` instead of `is_N_D_index`?

Reply:
I like that better, will implement